unit/ctest_skin_surf_from_ghost.c

```diff
@@ -0,0 +1,398 @@
+/**
```

Comment:
A comment for future work: although we do want to write unit tests in a way that allows to test more than one thing with the same code, writing a single test that tests too many things might be undesirable because it's harder to follow and harder to maintain. Consider that for the next unit test you create.